frontend/src/components/ModelBox/ModelConfiguration.tsx

```diff
@@ -67,7 +84,11 @@ function ModelConfiguration() {
 	return (
 		<div className="model-config-box">
 			{customChatModelConfigs.map((config) => (
-				<ModelConfigurationSlider key={config.id} config={config} />
+				<ModelConfigurationSlider
+					key={config.id}
+					config={config}
+					updateStateConfigValue={updateConfigValue}
```

Comment:
The name "updateStateConfigValue" is too specific for the ModelConfigurationSlider component, which should not know or care what any component that renders it is going to do with the changes.

Instead, name it "onConfigChanged" and the parent component will handle that change however it wishes.